sandbox/elephantsdream.ad.vtt.example

```diff
@@ -0,0 +1,278 @@
+WEBVTT

```

Comment:
I had them in the `build/demo-files` directory, but that didn't seem like the right place for them, particularly with that directory going away for V5. Sandbox seemed like the best place for now, but feel free to suggest somewhere else.